zoo-css-challenge/style.css

```
@@ -69,7 +73,7 @@ button {
 footer {
   background-color: #211a1d;
   color: #ffffff;
-  width: 100%;
+  width: 98%;
```

```suggestion
  width: 100%;
```
We don't need to make this change for any of the tasks, so we can change it back